gnocchi/cli/metricd.py

```diff
@@ -63,11 +68,13 @@ def run(self):
         while not self._shutdown.is_set():
             with utils.StopWatch() as timer:
                 self._run_job()
-            self._shutdown.wait(max(0, self.interval_delay - timer.elapsed()))
+            self._wake_up.wait(max(0, self.interval_delay - timer.elapsed()))
+            self._wake_up.clear()
```

Comment:
same thing.

Reply:
same thing as?

Reply:
as the "clear before we process it" comment.

Reply:
not sure I follow here… `_wake_up` is a `threading.Event` object :)

Reply:
ok nevermind. i was thinking wrong.